src/main/webapp/electionInfo.html

```diff
@@ -154,7 +158,7 @@ <h3 id="info-cards-title">What's on the ballot</h3>
                         <p class="referendum-description">
                           {{this.description}}
                         </p>
-                      </div>
+                      </div>	
```

Comment:
Nit: remove this random space. 

1) Random spaces are generally bad style
2) Small misc changes like this end up piling up on bigger PRs, and litter the PR to make it harder to review. 

I'm surprised the linter didn't auto-fix this actually.

Reply:
done